rest-api/image/controllers/image-controllers.js

```diff
@@ -1,18 +1,20 @@
+const db = require('../models/image-models')
```

Comment:
Should this be `dbImage`?

Reply:
it won't matter since there are no other imports in the moment